crates/router/src/core/payments/operations/payment_response.rs

```diff
@@ -1902,7 +1902,7 @@ async fn payment_response_update_tracker<F: Clone, T: types::Capturable>(
             status: api_models::enums::IntentStatus::foreign_from(
                 payment_data.payment_attempt.status,
             ),
-            return_url: router_data.return_url.clone(),
+            return_url: None,
```

Comment:
once verify that we are updating the `return_url` if it is sent in the confirm request. You can remove this field as well if it is not being used in other places

Reply:
yes we are updating the `return_url` if it i sent in confirm request.